app/views/members/_form.html.erb

```diff
@@ -39,12 +38,7 @@
     <%= f.text_area :bio %>
   </div>
 
-  <p style="width: 1px; height: 1px; overflow: hidden;">
-    Please leave following field blank:
-    <%= f.text_area :username %>
```

Comment:
You leave that in place (or wherever appropriate) and ignore it. The point is that only spam bots will fill it out.